wiki/WORKFLOW.md

```diff
@@ -0,0 +1,61 @@
+# Workflow
+
+This document provides an overview of the collaboration principles that drive our project. It includes information relative to the contribution workflow, the structure of meeetings and the organization of sprints and epics (iterations).
+
+> No document is auto-sufficient or frozen over time. This is also the case with the workflow of our team, which we will update as time goes by.
+
+## Contributing
+
+We follow an approach inspired by the [Rapid Cycle Workflow](https://gitlab.com/softeng-heigvd/teaching-heigvd-pdg-2020/guidelines/-/blob/master/WORKFLOW.md). We adapt it to match our specific use-case:
+
+- we're using GitHub, in order to centralize all of our services (issues, project management, milestones, hosting, continuous integration);
+- we want to make sysetmatic pull request reviews; and
+- we'll be using an external time tracking system.
+
+### Rapid Cycle Workflow
+
+1. Based on what was agreed on during meetings, file an issue for a task you'd like to accomplish. When choosing a task, choose something that you think should take you **less than 4 hours** to achieve. It's better to aim for a task that's too small rather than a task that's too big !
+	+ Use the dedicated issue template to file that task.
+	+ Fill in our time tracking system the expected duration of this task.
+2. Use your personal fork, and create a branch dedicated to the issue. Once you have your first commit ready, immediatly create a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
```

Comment:
I'd say something like "Create a DPR as soon as possible". I do not want to limit that to the first commit as one can be offline and still work (for example).

Reply:
Please let me know if 68a5127b6adadc2c651f2837d2008bb3a03daad5 sounds better ^^

Reply:
Perfect